cpp/include/cuspatial_test/vector_factories.cuh

```diff
@@ -164,10 +164,26 @@ class multipolygon_array {
   {
     auto [geometry_offsets, part_offsets, ring_offsets, coordinates] = arr.to_host();
 
-    return os << "Geometry Offsets:\n\t{" << geometry_offsets << "}\n"
-              << "Part Offsets:\n\t{" << part_offsets << "}\n"
-              << "Ring Offsets: \n\t{" << ring_offsets << "}\n"
-              << "Coordinates: \n\t{" << coordinates << "}\n";
+    auto print_vector = [&](auto const& vec) {
```

Comment:
Without these changes, I was seeing errors that no `operator<<` overload existed for `thrust::host_vector`.

Reply:
Did thrust remove this overload from host_vector?

Reply:
I wasn't aware that such an overload ever existed. I don't see that implemented: https://github.com/search?q=repo%3ANVIDIA%2Fcccl%20operator%3C%3C&type=code